### PR TITLE
MYST-564 Fix config directory in standalone artifacts

### PR DESCRIPTION
**Pull request by Waldz**

### CODE REVIEW

File: bin/client_package_standalone [hunk 1]
```diff
@@ -4,18 +4,37 @@
...
 # Put all files to package directory (to avoid preserving directory structure)
 DIR_TEMP=`mktemp -d ${DIR_BUILD}/${tempname}.XXXXXX`
-cp ${BINARY} ${DIR_TEMP}/
-cp -r "bin/client_package/config" ${DIR_TEMP}/
+cp -p ${BINARY} ${DIR_TEMP}/mysterium_client
+cp -rp "bin/common_package/" ${DIR_TEMP}/config
+cp -rp "bin/client_package/config/${OS}/" ${DIR_TEMP}/config 2>/dev/null || true
 
 # Tarball package directory
 tar -zcv -f ${PACKAGE_FILE} -C ${DIR_TEMP} .
```
Comment (tadovas) [inline bin/client_package_standalone]: Zip for windows? :)

Comment (Waldz) [reply bin/client_package_standalone]: Implemented

File: bin/client_package_standalone [hunk 2]
```diff
@@ -4,19 +4,47 @@
...
-cp ${BINARY} ${DIR_TEMP}/
-cp -r "bin/client_package/config" ${DIR_TEMP}/
+cp -p ${BINARY} ${DIR_TEMP}/mysterium_client
+cp -rp "bin/common_package/" ${DIR_TEMP}/config
+cp -rp "bin/client_package/config/${OS}/" ${DIR_TEMP}/config 2>/dev/null || true
 
 # Tarball package directory
-tar -zcv -f ${PACKAGE_FILE} -C ${DIR_TEMP} .
+if [ "$OS" == "windows" ]; then
+    PACKAGE_FILE="${DIR_BUILD}/`basename -s .exe ${BINARY}`.zip"
```
Comment (tadovas) [inline bin/client_package_standalone]: 👍 windows users happy :)

File: bin/client_package_standalone [hunk 3]
```diff
@@ -4,19 +4,47 @@
...
 
+source bin/helpers/output.sh
+
+OS=$1
+if [ -z "$OS" ]; then
+    print_error "Missing OS!"
+    exit 1
+fi
+if [ "$OS" != "linux" ] && [ "$OS" != "darwin" ] && [ "$OS" != "windows" ]; then
+    print_error "OS should be: linux, darwin, windows!"
```
Comment (zolia) [inline bin/client_package_standalone]: "OS should be: linux, darwin or windows", would skip "!".

File: bin/server_package_standalone [hunk 4]
```diff
@@ -4,19 +4,48 @@
...
 
+source bin/helpers/output.sh
+
+OS=$1
+if [ -z "$OS" ]; then
+    print_error "Missing OS!"
+    exit 1
+fi
+if [ "$OS" != "linux" ] && [ "$OS" != "darwin" ] && [ "$OS" != "windows" ]; then
+    print_error "OS should be: linux, darwin, windows!"
```
Comment (zolia) [inline bin/server_package_standalone]: "OS should be: linux, darwin or windows", would skip "!".

File: bin/client_package_standalone [hunk 5]
```diff
@@ -4,19 +4,47 @@
...
 set -e
 
+source bin/helpers/output.sh
+
+OS=$1
+if [ -z "$OS" ]; then
+    print_error "Missing OS!"
+    exit 1
+fi
+if [ "$OS" != "linux" ] && [ "$OS" != "darwin" ] && [ "$OS" != "windows" ]; then
```
Comment (zolia) [inline bin/client_package_standalone]: theres also more concise version for this check: 
if [[ ! "$OS" =~ ^(linux|darwin|windows)$ ]]; then

Comment (Waldz) [reply bin/client_package_standalone]: Good one. Improved

File: bin/server_package_standalone (hunk 4 above)
Comment (Waldz) [reply bin/server_package_standalone]: Improved

File: bin/client_package_standalone (hunk 3 above)
Comment (Waldz) [reply bin/client_package_standalone]: Improved

**Pull request merged**